package.json

```diff
@@ -353,14 +353,14 @@
     "yocto-queue": "^0.1.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.9",
-    "@babel/preset-env": "^7.16.11",
-    "babel-loader": "^8.2.4",
+    "@babel/core": "^7.17.12",
+    "@babel/preset-env": "^7.17.12",
```

Comment:
What is our approach in bumping packages?

Reply:
I would assume alot of packages are behind and a cleanup task needs to be preformed to bump the ones with updates and fix any potential changes without doing anything major